src/rule-unused-fields.js

```diff
@@ -206,4 +287,22 @@ function rule(context) {
   };
 }
 
-module.exports = rule;
+module.exports = {
+  meta: {
```

Comment:
Need to expand this with `meta` field because the rule now accepts parameter for the whitelisting function name to make it configurable.